src/main/java/org/testobject/test/LogentriesResultReporter.java

```diff
@@ -0,0 +1,69 @@
+package org.testobject.test;
```

Comment:
`test` is a pretty nonspecific name since so many of our tools are for tests. Could we call it something like "thirdparty" or "integrations"?